### PR TITLE
Add organization philosophy

### PR DESCRIPTION
**Pull request by tmjd**
Thanks Casey.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,13 +70,51 @@ Then simply inspect the results (`git status`, `git diff`, etc.) and commit.
 
 ## Navigation & Sidebar
 
-The docs (currently) are split into 4 sections:
+The docs (currently) are split into 3 main sections:
 
-- what
```
Comment (djosborne) [inline README.md]: Can you restore this as "Introduction"

It is:

Landing page for new users covering Calico's purpose and high-level topics.

Comment (djosborne) [inline README.md]: I actually have some new material planned for this section in the coming weeks :smile:

Comment (tmjd) [inline README.md]: Nice to hear.  It will help us all keep in line with organization in the docs.

**Comment by caseydavenport** [comment]
LGTM

**Pull request merged**